pkg/registry/scripts/projects.js

```diff
@@ -34,10 +35,54 @@
         }
     ])
 
+    .factory("projectUtil", [
+        function() {
+            return {
+                formatMembers: function (members, kind) {
+                    var mlist = "";
+                    if (!members || members.length === 0)
+                        return mlist;
+                    if (members.length <= 3) {
+                        for (var i = members.length - 1; i >= 0; i--) {
```

Comment:
We shouldn't be declaring variables inside of an if statement. Unfortunately that's how javascript is. Since |i| is valid across the whole function, lets keep it declared outside of any if statement or other scope element.